addons/im_livechat/controllers/main.py

```diff
@@ -149,10 +149,24 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
         if previous_operator_id:
             previous_operator_id = int(previous_operator_id)
 
+        return request.env["im_livechat.channel"].with_context(lang=False).sudo().browse(channel_id)._get_livechat_session(
+            anonymous_name,
+            previous_operator_id=previous_operator_id,
+            chatbot_script=session_data['chatbot_script'],
+            user_id=session_data['user_id'],
+            country_id=session_data['country_id']
+        )
+
+    @http.route('/im_livechat/create_livechat_channel', type="json", auth='public', cors="*")
+    def create_livechat_channel(self, channel_id, anonymous_name, uuid=None, previous_operator_id=None, chatbot_script_id=None):
```

Comment:
It looks dangerous to allow passing `uuid` as parameter and using it as a create value.

2 different channels should not be allowed to have the same `uuid`, do we have unique constraint?